daweb/js2/organizace-bundle/vite.md

```diff
@@ -0,0 +1,103 @@
+## Vite
+
+Ve chvili, kdy máme založený projekt, budeme potřebovat nástroj, který vezme náš kód spolu s použitými knihovnami a zabalí je všechny do jednoho velkého JavaScriptového souboru. Ten si pak prohlížeč může stáhnout celý najednou a nemusí tak čekat na spoustu malých souborů jednotlivě po sobě. Nástrojům, které toto pro nás dělají se říká _bundlery_. Existuje jich několik různých. Jeden z nejpopulárnější se jmenuje [_Webpack_](https://webpack.js.org/). My v tomto kurzu budeme používat modernější bundler jménem [_Vite_](https://vitejs.dev/).
+
+Nejdříve musíme do našeho projektu Vite nainstalovat. To provedeme příkazem
+
+```sh
+$ npm install -D vite
```

Comment:
Navrhuji v akademii vůbec neučit devDependencies.

![image](https://github.com/Czechitas-podklady-WEB/daweb-vyuka/assets/1045362/29bbea16-6a67-48df-be8f-729b4d83a711)


Tady je to stejně špatně použité. `dayjs` a `validator` jsou také `devDependencies`.